qiskit_ibm_runtime/qpy/exceptions.py

```diff
@@ -12,6 +12,7 @@
 
 """Exception for errors raised by the pulse module."""
 from typing import Any
+from qiskit.exceptions import QiskitWarning
```

Comment:
This feels ugly, but a workaround to make this import work in both 0.45 and 1.0 could be aliasing `UserWarning`:

```suggestion
try:
    from qiskit.exceptions import QiskitWarning
except ImportError:
    QiskitWarning = UserWarning
```